fluent-bit/fluent-bit-go-loki/README.md

```diff
@@ -0,0 +1,54 @@
+# fluent-bit loki output plugin
+
+This plugin works with fluent-bit's go plugin interface. You can use fluent-bit loki to ship logs into grafana datasource with loki.
```

Comment:
Shouldn't this be 
`It allows to fluent-bit to ship logs into Loki, for use with e.g. Grafana?`